kubernetes/base/serviceaccount.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
   name: {{SERVICE_NAME}}
   namespace: {{NAMESPACE}}
   annotations:
-    eks.amazonaws.com/role-arn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/interop-iam-service-{{LOWERCASE_ENV}}-{{SERVICE_NAME}}
\ No newline at end of file
+    eks.amazonaws.com/role-arn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/{{SERVICE_NAME}}-{{LOWERCASE_ENV}}
```

Comment:
This change is in the base so it would apply to all services, but we don't want that. Some of them are still using the old naming convention 🥲 
I suggest creating a `serviceaccount.yaml` file with the new naming convention in each env for this service and add it to `kustomization.yaml`

Reply:
sorry, you're right, I changed the wrong file 🥲